src/sql/ResourceSelector.tsx

```diff
@@ -99,7 +100,7 @@ export function ResourceSelector(props: ResourceSelectorProps) {
   };
 
   return (
-    <InlineField label={props.label} labelWidth={props.labelWidth} tooltip={props.tooltip}>
+    <InlineField label={props.label} labelWidth={props.labelWidth} tooltip={props.tooltip} hidden={props.hidden}>
```

Comment:
I know I've said it before, but I'd replace the async call with the useAsyncFn. It's really neat. Did this recently [here](https://github.com/grafana/cloudwatch-private/blob/1660d063bb7a5e86e9ccebd488b4cead08c27114/public/app/plugins/datasource/cloudwatch/components/MetricStatEditor/FilterItem.tsx#L63). It's really neat.